scripts/skills/perks/perk_rf_offhand_training.nut

```diff
@@ -21,34 +20,19 @@ this.perk_rf_offhand_training <- ::inherit("scripts/skills/skill", {
 
 	function isHidden()
 	{
-		return this.m.IsSpent && !this.m.IsFreeUse;
+		return !this.m.IsFreeUse;
 	}
 
 	function getTooltip()
 	{
-		local tooltip = this.skill.getTooltip();
-
-		if (!this.m.IsSpent)
-		{
-			tooltip.push({
-				id = 10,
-				type = "text",
-				icon = "ui/icons/special.png",
-				text = "The first swap of an offhand item with a weight less than " + ::MSU.Text.colorRed(-this.m.StaminaModifierThreshold) + " costs no Action Points"
-			});
-		}
-
-		if (this.m.IsFreeUse)
-		{
-			tooltip.push({
-				id = 10,
-				type = "text",
-				icon = "ui/icons/special.png",
-				text = "The first use of a tool in the offhand this turn costs no Action Points"
-			});
-		}
-
-		return tooltip;
+		local ret = this.skill.getTooltip();
+		ret.push({
+			id = 10,
+			type = "text",
+			icon = "ui/icons/special.png",
+			text = ::Reforged.Mod.Tooltips.parseString("The first use of any offhand item weighing less than " + ::MSU.Text.colorNegative(-this.m.StaminaModifierThreshold) + " this [turn|Concept.Turn] costs no [Action Points|Concept.ActionPoints]")
```

Comment:
Now that this extends to all skills provided by that offhand item, we should mention that it in the tooltip explicitely.
`The first skill use of any offhand item [...]`

Reply:
I think that's not necessary and makes it too mechanical in my opinion. I prefer the current wording in the PR i.e. `the first use of any offhand item` - the fact that it refers to that item's skills is implicit.